play/src/pages/pregame/HowToPlay.tsx

```diff
@@ -35,69 +36,82 @@ const HowToPlaySwiper = styled(Swiper)({
-          }}
-        >
-          {t('joingame.howtoplay.title')}
-        </Typography>
-        <Box style={{ width: '100%' }}>
-          <HowToPlaySwiper
-            modules={[Pagination]}
-            slidesPerView={1}
-            pagination={{
-              el: '.swiper-pagination-container',
-              bulletClass: 'swiper-pagination-bullet',
-              bulletActiveClass: 'swiper-pagination-bullet-active',
-              clickable: true,
-              renderBullet(index, className) {
-                return `<span class="${className}" style="width:20px; height:6px; border-radius:0"></span>`;
-              },
-            }}
-          >
-            <SwiperSlide>
-              <HowToPlaySlide0Content />
-            </SwiperSlide>
-            <SwiperSlide>
-              <HowToPlaySlide1Content />
-            </SwiperSlide>
-            <SwiperSlide>
-              <HowToPlaySlide2Content />
-            </SwiperSlide>
-            <SwiperSlide>
-              <HowToPlaySlide3Content />
-            </SwiperSlide>
-            <SwiperSlide>
-              <HowToPlaySlide4Content />
-            </SwiperSlide>
-          </HowToPlaySwiper>
-          <PaginationContainerStyled
-            className="swiper-pagination-container"
-            style={{ paddingTop: `${theme.sizing.largePadding}px` }}
-          />
-        </Box>
-        <Typography
-          variant="h4"
-          sx={{
-            color: `${theme.palette.primary.main}`,
-            fontWeight: 400,
-            textAlign: 'center',
-            paddingBottom: `${theme.sizing.mediumPadding}px`,
+      <Typography
+        variant="h2"
+        sx={{
+          textAlign: 'center',
+          paddingTop: `${theme.sizing.mediumPadding}px`,
+        }}
+      >
+        {t('howtoplay.title')}
+      </Typography>
+      <StackContainer
+        style={{ position: 'absolute', justifyContent: 'center' }}
+      >
+        <HowToPlaySwiper
+          modules={[Pagination]}
+          slidesPerView={1}
+          pagination={{
+            el: '.swiper-pagination-container',
+            bulletClass: 'swiper-pagination-bullet',
+            bulletActiveClass: 'swiper-pagination-bullet-active',
+            clickable: true,
+            renderBullet(index, className) {
+              return `<span class="${className}" style="width:20px; height:6px; border-radius:0"></span>`;
+            },
           }}
         >
-          {t('joingame.howtoplay.description')}
-        </Typography>
+          <SwiperSlide>
+            <HowToPlaySlide0Content />
+          </SwiperSlide>
+          <SwiperSlide>
+            <HowToPlaySlide1Content />
+          </SwiperSlide>
+          <SwiperSlide>
+            <HowToPlaySlide2Content />
+          </SwiperSlide>
+          <SwiperSlide>
+            <HowToPlaySlide3Content />
+          </SwiperSlide>
+          <SwiperSlide>
+            <HowToPlaySlide4Content />
+          </SwiperSlide>
```

Comment:
hmm, maybe have an array that includes all the `HowToPlaySlideXContent` components and then use `map` to create this. Not important, but can be a good practice to reduce the code while making it easier to add any new content as instead of copy pasting code here, you just need to add the new content in the array

Reply:
nice, map statement has been added with <HowToPlaySlideXContent /> in an array